ckan/templates/package/search.html

```diff
@@ -4,7 +4,8 @@
 {% block subtitle %}{{ _("Datasets") }}{% endblock %}
 
 {% block breadcrumb_content %}
-  <li class="active">{{ h.nav_link(_('Datasets'), controller='package', action='search', highlight_actions = 'new index') }}</li>
+  {% set type_text = dataset_type.title() %}
+  <li class="active"><a href="{{ dataset_type }}">{{ _(type_text) }}</a></li>
```

Comment:
should be possible to generate these urls with nav_link or url_for. They are registered  as `'%s_search' % package_type` https://github.com/ckan/ckan/blob/master/ckan/lib/plugins.py#L104